doc/python_api/rst/bge_types/bge.types.KX_Scene.rst

```diff
@@ -143,10 +143,15 @@ base class --- :class:`PyObjectPlus`
 
       :type: Vector((gx, gy, gz))
 
-   .. attribute:: lodGlobalScale
+   .. method:: setLodScale(pythonObjectList, scale)
```

Comment:
I'm not sure to understand the use here. If the user have a list of wanted objects why not iterating on it and modify the lodManager.distanceScale value ?